addon/templates/components/frost-bookends.hbs

```diff
@@ -0,0 +1,54 @@
+{{! Template for the frost-bookends component }}
+<div
+  class='{{concat css '-header'}}'
+  data-test={{hook (concat hookPrefix '-headerWrapper') hookQualifiers}}
+>
+  {{component header
+    hook=(concat hookPrefix '-header')
+    hookQualifiers=hookQualifiers
+  }}
+
+  {{#if isContentScrolledBeneathHeader}}
+    <div class='{{concat css '-content-beneath-header-effect'}}'
+      data-test={{hook (concat hookPrefix '-contentBeneathHeaderEffect') hookQualifiers}}
+    >
+    </div>
+  {{/if}}
+</div>
+
+{{#frost-scroll
+  class=(concat css '-scroll')
+  hook=(concat hookPrefix '-scroll')
+  hookQualifiers=hookQualifiers
+  onScrollDown=(action 'showContentBeneathHeaderEffect')
+  onScrollUp=(action 'showContentBeneathFooterEffect')
+  onScrollYEnd=(action 'hideContentBeneathFooterEffect')
+  onScrollYStart=(action 'hideContentBeneathHeaderEffect')
+}}
+  <div
+    class='{{concat css '-content'}}'
+    data-test={{hook (concat hookPrefix '-contentWrapper') hookQualifiers}}
+  >
+    {{component content
```

Comment:
Hmm. I was actually running into that myself when consuming this. Was gonna go down the route of a computed property for the content, but I don't mind that at all.